src/feature_alert/hooks.es

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react';
+import keymanager from 'nebenan-helpers/lib/keymanager';
+import { invoke } from 'nebenan-helpers/lib/utils';
+
+export const useEscHandler = (callback) => useEffect(() => keymanager('esc', callback), []);
+export const useMisClickHandler = (ref, callback) => useEffect(() => {
+  const handler = (event) => {
+    if (!ref.current) return;
+    if (!ref.current.contains(event.target)) invoke(callback, event);
+  };
+  document.addEventListener('click', handler);
+
+  return () => document.removeEventListener('click', handler);
+}, []);
```

Comment:
callback in deps or useRef for callback?

Reply:
putting it in deps would make it necessary to wrap it in useCallback also, a bit tedious